app/controllers/time_logs_controller.rb

```diff
@@ -0,0 +1,54 @@
+class TimeLogsController < ApplicationController
+  before_action :set_time_log, only: [:show, :edit, :update, :destroy]
+
+  def new
+    find_project
+    @timelog = TimeLog.new
+  end
+
+  def edit
+    find_project
+  end
+
+  def create
+    @timelog = TimeLog.new(timelog_params)
+    @timelog.project_id = params[:project_id]
```

Comment:
Why are we setting the project_id from params here ?
Try to create time log like:
`@project.time_logs.new(timelog_params)`